elasticsearch/examples/multi/values.yaml

```diff
@@ -0,0 +1,25 @@
+# The data nodes
+es-data:
+  <<: *defaults # include with the default settings
+  nodeGroup: "data"
+
+  roles:
+    master: "false"
+    ingest: "true"
+    data: "true"
+
+# The master nodes
+es-master:
+  <<: *defaults # include with the default settings
+  nodeGroup: "master"
+
+  roles:
+    master: "true"
+    ingest: "false"
+    data: "false"
+
+# To define common settings, it may be useful to use YAML anchors to use the same 
```

Comment:
In helm 2.14.3, the anchor has to be defined BEFORE using it, so moving this part on top of the file works.